sos/plugins/sapnw.py

```diff
@@ -99,10 +99,10 @@ def collect_list_dbs(self):
             suggest_filename="SAPDatabases"
         )
 
-        if not db_out:
+        if db_out['status'] != 0:
             return
 
-        dbl = open(db_out, "r").read().splitlines()
+        dbl = open(db_out['filename'], "r").read().splitlines()
```

Comment:
Same here. `collect_cmd_output` already returns the command's output in the dict.